### ISSUE TITLE
docker-compose build fails

### ISSUE THREAD

**Issue by TomCJones** [decentralized-identity/sidetree#853]
with the message 
Cannot locate specified Dockerfile: docker/sidetree-bitcore/Dockerfile
perhaps bitcoin was meant to be created by ION??
That that leads me to the next question, I thought sidetree was DLT agnostic - why is bitcoin required?

**Comment by OR13** [comment]
yes, this is due to the unfortunate module structure of ION and Sidetree Core.... We are working on a fix for this, but its unfortunately required substantial surgery... and we are also having trouble getting some of the core deps to run in the browser.... 

I expect this will be solved as sidetree matures.

**Comment by OR13** [comment]
@tplooker I think maybe you have fixed this somewhere else?
